testing/widgets/list/ListSelection.test.js

```diff
@@ -17,6 +18,8 @@ runManualTest('List', 'ListSelection', ['jQuery', 'React', 'Vue', 'Angular'], (t
   test('List Selection', async (t) => {
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
+    await t.click(Selector(`.${CHECKBOX_CONTAINER_CLASS}`).nth(-1));
```

Comment:
i checked 'selectByClick' so test behaviour stays the same as before (  selectByClick's default value used to be true )